pages/osx/defaults.md

```diff
@@ -0,0 +1,19 @@
+# defaults
+
+> Access OS X user defaults
+
+- read all defaults
+
+`defaults read -app {{app-name}}`
+
+- read specific defaults
+
+`defaults read -app {{app-name}} {{key}}`
+
+- write key value
+
+`defaults write -app {{app-name}} {{key}} {{-type}} {{value}}`
+
+- delete all defaults
+
+`defaults write -app {{app-name}}`
```

Comment:
Yes, Missed that error. Thanks for pointing out.
Corrected now and update has been committed.